fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -1596,4 +1611,13 @@ abstract class BaseBrowserFragment :
 
         return isValidStatus && isSameTab
     }
+
+    private suspend fun provideAddons(): List<Addon> {
+        return withContext(IO) {
+            // We deactivated the cache to get the most up-to-date list of add-ons to match against.
+            // As this will be used to install add-ons from AMO
```

Comment:
nit:

```suggestion
            // We deactivated the cache to get the most up-to-date list of add-ons to match against
            // as this will be used to install add-ons from AMO.
```